packages/apps/plugins/plugin-thread/src/ThreadPlugin.tsx

```diff
@@ -371,7 +365,6 @@ export const ThreadPlugin = (): PluginDefinition<ThreadPluginProvides> => {
                           threads={threads}
                           detached={detached}
                           currentId={attention.has(qualifiedSubjectId) ? state.current : undefined}
-                          context={context}
```

Comment:
(realizing this functionality was experimental and we don't need it fully working to land this) did you have a place for how this might be restored? if so, perhaps a todo would be useful?

Reply:
Context was both never resolving, and also the prop was never being used so I axed it, admittedly without getting to the bottom of what it was used for in the past.

Reply:
it was used for some of the AI demos which depended on knowing what you were currently looking at when you sent the message. Make sure @richburdon sees this, he may prefer is stay around commented out because hooking some of that back up is still on his todo list